# Methods

Once you have settled on a group that you are interested in studying you need to find out
if there are data to help you answer your research questions.
We'll be using genome sequence data to determine the species relationships in your group.
Read the following to learn more about sequencing and molecular phylogenetics:

* [2nd generation sequencing](https://www.youtube.com/watch?v=fCd6B5HRaZ8)
* [Learn more about how molecular data is used in phylogenetics](https://www.ebi.ac.uk/training-beta/online/courses/introduction-to-phylogenetics/what-is-phylogenetics/why-use-molecular-data/).
* [Genomes ch. 16](https://www.ncbi.nlm.nih.gov/books/NBK21122/)

To acquire molecular data to compare the species in your group 
you either need to do a lot of sequencing, or find sequencing someone else has done.
Becuase sequencing takes time and money we'll try using data that is already available
and just hasn't been analyzed to answer the questions you are interested in.

1. Go to the [European Nucleotide Archive](https://www.ebi.ac.uk/ena/browser/advanced-search) *add background*
2. Select: Data type: Raw Reads
3. Query: pick taxonomy (include subordinate)
4. Add to query box: `AND instrument_platform="ILLUMINA" AND (library_strategy="RNA-Seq" OR library_strategy="WGS")`
5. Fields: base_count, fastq_bytes, fastq_ftp, read_count, scientific_name, tax_id
6. Click Download report tsv

Now you have a list of available data and relevant information to get it.
You can open this file in Excel or Google Sheets.

It might look like you have a lot of data, but it's actually hard to tell from this page.
You might have a lot of data for one or a few species and not enough to analyze your whole group.
You could count up how much data you have by hand, but that would take a lot of time
if you have a lot of samples.
Sort your data by scientific_name.
Try to get 10 times as many bases (base_count) as the size of the genome for each species.
Select the samples you want and save those in a separate sheet.
Split the ftp column so you have both links.
